### PR TITLE
Charter roster management + localStorage save/load charter [ch2416]

### PR DESCRIPTION
**Pull request by tobek**

### CODE REVIEW

**Comment by walfly** [comment]
`cd packages/components && yarn test -u`

**Comment by walfly** [comment]
I think it might make sense to pass in the full roster component from the plugin.

**Comment by walfly** [comment]
oh woops wrong button

File: packages/newsroom-manager/src/RosterMember.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+export interface RosterMemberState {
+  newUserAddress?: EthAddress;
+}
+
+const Wrapper = styled.div`
+  padding: 20px 0;
+  border-top: 1px solid ${colors.accent.CIVIL_GRAY_4};
+`;
+
+const DisplayName = FormSubhead.extend`
```
Comment (walfly) [inline packages/newsroom-manager/src/RosterMember.tsx]: no extend https://github.com/styled-components/styled-components/issues/1546

Comment (tobek) [reply packages/newsroom-manager/src/RosterMember.tsx]: regex'd all of them out from whole repo

**Pull request merged**